### ISSUE TITLE
Luminosity ratio of ‘This is the subtitle’ text is less than 4.5:1.

### ISSUE THREAD

**Issue by 232amansingh** [microsoft/BotFramework-WebChat#5746]
### Is it an issue related to Adaptive Cards?

Yes, this is related to Adaptive Card. However, it does not repro on Adaptive Cards Designer.

### What is the PWD impact?

When the “This is the subtitle” text has a luminosity contrast ratio less than 4.5:1, users with low vision or those in bright or high‑contrast environments may struggle to read it. This reduces readability, causes eye strain, and excludes users who depend on sufficient text contrast.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Repro Steps: ​           
1. Open URL: Web Chat in edge browser.
2. Ask query 'Carousel'.
3. Check the CCA of ‘This is the subtitle’ text.
4. Observe the issue that Luminosity ratio of ‘This is the subtitle’ text is less than 4.5:1.

### What do you expect?

Luminosity ratio of ‘This is the subtitle’ text should be equal or greater than 4.5:1.

### What actually happened?

Luminosity ratio of ‘This is the subtitle’ text is less than 4.5:1.

### Do you have any screenshots or recordings to repro the issue?

<img width="1919" height="980" alt="Image" src="https://github.com/user-attachments/assets/29cc503d-1af3-433f-9966-64173a498a1d" />

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

### Adaptive Card JSON

```json

```
